tests/acceptance/setup/test.js

```diff
@@ -21,14 +22,17 @@ describe('Generate a dummy report for testing', function() {
     let browser, mugshot, webdriverioInstance;
 
     before(function(done) {
-      webdriverioInstance = webdriverio.remote(BROWSER_OPTIONS).init()
-        .url(URL)
-        .then(function() {
-          browser = new WebdriverIOAdapter(webdriverioInstance);
-          mugshot = new Mugshot(browser, MUGSHOT_OPTIONS);
+      cleanUp('visual-report', function() {
```

Comment:
This is the test file that creates the report.

I am cleaning up in the test file which tests the reporter:
https://github.com/uberVU/mocha-mugshot-reporter/blob/48-clean-after-tests/tests/acceptance/test.js#L75